Description: On mips{,el}, do not pass -mabi=n32 flag.
 The mips{,el} ports use the O32 ABI.
 See https://www.linux-mips.org/wiki/MIPS_ABI_History for more details on ABIs.
Bug: https://sourceforge.net/p/math-atlas/support-requests/1053/
Applied-Upstream: 3.10.4
Last-Update: 2017-08-25
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CONFIG/src/SpewMakeInc.c
+++ b/CONFIG/src/SpewMakeInc.c
@@ -402,7 +402,7 @@ char *GetPtrbitsFlag(enum OSTYPE OS, enu
    if (MachIsIA64(arch))
       return(sp);
    if (MachIsMIPS(arch))
-      return((ptrbits == 64) ? "-mabi=64" : "-mabi=n32");
+       return(sp);
    if (MachIsS390(arch))
       return((ptrbits == 64) ? "-m64" : "-m31");
    if (OS == OSAIX)
--- a/CONFIG/src/probe_comp.c
+++ b/CONFIG/src/probe_comp.c
@@ -596,7 +596,7 @@ char *GetPtrbitsFlag(enum OSTYPE OS, enu
    if (MachIsIA64(arch))
       return(sp);
    if (MachIsMIPS(arch))
-      return((ptrbits == 64) ? "-mabi=64" : "-mabi=n32");
+       return(sp);
    if (MachIsS390(arch))
       return((ptrbits == 64) ? "-m64" : "-m31");
    if (OS == OSAIX)
